pandunia-loge.csv

```diff
@@ -2286,11 +2285,11 @@ lote|||por:spa:lote, eng:fra:lot|batch (lot)||lote (grupo)||||||||||||||erä (sa
 lotra|bio|Lutrinae|eng:otter + fra:loutre, spa:tgl:lutrino, por:lontra + rus:выдра (vydra)|otter|loutre|nutria|lontra|выдра||獭|獺||||||||||wydra
 lou|||zho:漏 (lòu), yue:漏 (lau6), jpn:漏 (rō), 루 (ru)|leak||chorrear|vazar|протекать (просочиться)||漏|漏る||||||||liki|vuotaa|przeciekać, ciec
 lou sang||||bleed||sangrar|||||||||||||sangi|vuotaa verta|krwawić
-luga|rang||zho:绿 (lǜ), yue:綠 (luk6), vie:lục, kor:록/녹 (rok/nok), jpn: 緑色 (ryokushoku)|green|vert|verde|verde|зелёный|أَخْضَر|绿色|緑 (緑色)|녹색|xanh (lục)|हरा|সবুজ|hijau|kijani||verda|vihreä|zielony
-luga melon|bio|Cucumis melo, Inodorus||honeydew (green melon)||melón verde|||||ハネデューメロン||||||||||melon miodowy
-luka|numbe|6|zho:六 (liù), jpn:六 (roku), kor:육/륙 (六, yuk/ryuk), vie:lục|six (6)|six (6)|seis (6)|seis (6)|шесть (6)||六 (6)|六 (6)|||||||||kuusi (6)|sześć (6)
-luka fase||||hexahedron (cube)||hexaedro (cubo)||||||||||||||kuutio (kuusitahokas)|sześcian (kostka)
-luka gona||||hexagon||hexágono||||||||||||||kuusikulmio|sześciokąt (sześciobok)
+luka|rang||zho:绿 (lǜ), yue:綠 (luk6), vie:lục, kor:록/녹 (rok/nok), jpn: 緑色 (ryokushoku)|green|vert|verde|verde|зелёный|أَخْضَر|绿色|緑 (緑色)|녹색|xanh (lục)|हरा|সবুজ|hijau|kijani||verda|vihreä|zielony
+luka melon|bio|Cucumis melo, Inodorus||honeydew (green melon)||melón verde|||||ハネデューメロン||||||||||melon miodowy
+loka|numbe|6|zho:六 (liù), jpn:六 (roku), kor:육/륙 (六, yuk/ryuk), vie:lục|six (6)|six (6)|seis (6)|seis (6)|шесть (6)||六 (6)|六 (6)|||||||||kuusi (6)|sześć (6)
```

Comment:
'loka' already means location.

We could still do one more iteration of numbers words: un, du, sam, char, tan, sis, chi, bal, nau, des...

Reply:
oh, I forgot about 'location'.  yes, a different root for "six" is probably the easiest way to resolve this conflict.

I do think that **chatre** and **penta** were a bit too long.  but I think four letters is an acceptable length for these (so I would keep the **t** in **bati** and include the final vowel on **tano**).  I also think **tri** is still better than **san**, since it combines European with Indo-Aryan.  **tisa** was nice because it added some Arabick influence, but **nau** is more internacional, so either makes sense to me.

Reply:
also, as a nitpick, "six" and seven are kind of similar in this set.  if "seven" is going to have an **i** in it (**chi** or **chiti**), it mite make sense to  give "six" an **e** to distinguish them (so **ses**).  otherwise if "seven" stays as **sete**, then six should be **sis**.

Reply:
Good points! I think that the Indo-European languages are maybe over-represented in the numerals. **tano** (5), **cheti** (7) and **tisa** (9) and would be good because they make the mix more evenly distributed.